infrastructure/ecs.tf

```diff
@@ -22,9 +25,72 @@ module "ndr-ecs-container-port-ssm-parameter" {
   source              = "./modules/ssm_parameter"
   name                = "container_port"
   description         = "Docker container port number for ${var.environment}"
-  resource_depends_on = module.ndr-ecs-fargate
-  value               = module.ndr-ecs-fargate.container_port
+  resource_depends_on = module.ndr-ecs-fargate-app
+  value               = module.ndr-ecs-fargate-app.container_port
   type                = "SecureString"
   owner               = var.owner
   environment         = var.environment
+}
+
+module "ndr-ods-update-fargate" {
```

Comment:
Can we rename this to `ndr-ecs-fargate-ods-update` please to match the name above